config.yml

```diff
@@ -0,0 +1,42 @@
+databricks_pipelines:
+    Ontwikkel:
+      catalog: "dpcv_dev"
+      schema: "oor"
+      storage_account_root_path: "abfss://landingzone@stlandingdpcvontweu01.dfs.core.windows.net"
+      device_id: "Luna"
+      ckpt_frames_relative_path: checkpoints/_checkpoint_frames
```

Comment:
What does ckpt stands for?

Reply:
checkpoints for the frames and detections metadata that are used by the autoloader to keep track on what to add to the tables